src/components/app/MenuButtons.css

```diff
@@ -150,3 +150,8 @@
   position: relative;
   bottom: 1px;
 }
+
+.menuButtonsOpenMetainfo {
+  width: 150px;
```

Comment:
The "Profile Info" button is the only button in the header that doesn't let its size be determined through the size of its contents.

The "Upload" / "Permalink" button is odd since it has a few different set of texts determining the sizing, so it's a bit wider than all the rest, but it's still sized according to the larger bit of text. This oddness is caused by the special animation that gets triggered when you upload profiles.

> <img width="426" alt="image" src="https://user-images.githubusercontent.com/1588648/47677888-556f5a00-db8e-11e8-9f42-e3c90502b080.png">

The upload button is also more dramatic when it's colored bright green:

> <img width="283" alt="image" src="https://user-images.githubusercontent.com/1588648/47678150-1261b680-db8f-11e8-940b-c1e33f2ae519.png">

What do you think about moving this to the right of the Upload/Permalink button? I think users are probably accustomed to the upload button being to the far left of that list. Then it will continue to be the only oddly sized button, and the Profile Info button can be sized according to its contents.

e.g to this:

> <img width="402" alt="image" src="https://user-images.githubusercontent.com/1588648/47678108-f6f6ab80-db8e-11e8-8cf6-c0dfeab457bb.png">

Reply:
The `Profile info` button will be replaced by #1055 and a meatball button. Those two issues got split up as we had separate discussions about it as well as keep the scope small.

This is why I would suggest to not put too much work into this right now. 
What do you think @gregtatum @mstange ?

Reply:
I don't see anything about a meatball button in #1055. And fixing this issue should only take two very simple modifications, so I think it's still worth doing. (Remove the width property and swap the order with `ProfileSharingCompositeButton` in the JSX.)